manage/installing/installation.rst

```diff
@@ -125,11 +125,11 @@ Download the latest Plone unified installer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Download from `the plone.org download page <https://plone.org/download>`_ to your server using wget command. Curl also works.
```

Comment:
@jensens Got catch ! May I offer a other solution ? 

The reason is I want to avoid that we have to fix all this tiny links all the time, yes there is also a plugin to do so, but still not perfect :)
We will face this more often and also with other releases like Plone 5, for example.

Since some people are not always able to use the latest version, maybe we could add a link with a big disclaimer to https://plone.org/download ?
Something like "you always should use the latest release, but if for some reason you need a older version, please go to https://plone.org/download/old and choose the version you need" or so which is a link to https://launchpad.net/plone/+download ?

@esteele @smcmahon @tkimnguyen @3dogMcNeill @polyester what do you think ? I included all of you, because if we do so that also should go on to new and old plone.org